composer/plugins/gcs_plugin/hooks/gcs_hook.py

```diff
@@ -0,0 +1,530 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import re
+
+from airflow.contrib.hooks.gcp_api_base_hook import GoogleCloudBaseHook
+from airflow.exceptions import AirflowException
+from apiclient.discovery import build
+from apiclient.http import MediaFileUpload
+from googleapiclient import errors
+
+
+class GoogleCloudStorageHook(GoogleCloudBaseHook):
```

Comment:
I'm a bit confused. Are these files copied from the Airflow repository? If so, could we provide download instructions rather than copying them here?

Reply:
Correct, both hook and operator are from the Airflow master branch (1.10), I just copied this snapshot version to avoid breaking the code with future changes in the Airflow project.
Let me know if there is a more elegant way (I guess can give download instructions from a specific git commit from the Airflow repo).

Reply:
OK, I removed these and added instructions to download with wget from 1.10-stable branch.

Reply:
Please remove the operator as well.

If we cannot, please retain the original license headers and move to a `third_party/` directory per go/thirdparty#projects